apps/docs/src/guide/wallets/encrypting-and-decrypting.md

```diff
@@ -18,10 +18,16 @@ To decrypt the JSON wallet and retrieve your private key, you can call `fromEncr
 
 Here is an example:
 
-<<< @/../../docs-snippets/src/guide/wallets/encrypting-and-decrypting-json-wallets.test.ts#encrypting-and-decrypting-json-wallets-2{ts:line-numbers}
+<<< @/../../docs-snippets2/src/wallets/encrypting-and-decrypting-json-wallets-two.ts#encrypting-and-decrypting-json-wallets-2{ts:line-numbers}
 
 In this example, `decryptedWallet` is an instance of [`WalletUnlocked`](../../api/Account/WalletUnlocked.md) class, now available for use.
 
 ## Important
 
 Remember to securely store your encrypted JSON wallet and password. If you lose them, there will be no way to recover your wallet. For security reasons, avoid sharing your private key, encrypted JSON wallet or password with anyone.
+
+## Full Example
+
+For a full example of how to encrypt and decrypt a JSON wallet, see the snippet below:
+
+<<< @/../../docs-snippets2/src/wallets/encrypting-and-decrypting-wallets.ts#full{ts:line-numbers}
```

Comment:
A little bit similar to what we have on https://github.com/FuelLabs/fuels-ts/pull/3365/files#r1833707433

The code from `encrypting-and-decrypting-json-wallets-2` was added to `encrypting-and-decrypting-json-wallets-1` in order for the full example snippets to be complete.

Reply:
Good point, I've removed it 1a7148913